chromium_src/services/network/resolve_host_request.cc

```diff
@@ -3,10 +3,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#define BRAVE_START                                   \
-  if (internal_request_->GetTextResults()) {          \
-    response_client->OnTextResults(                   \
-        internal_request_->GetTextResults().value()); \
+#define BRAVE_START                                                       \
```

Comment:
the define name is too generic. let's rename it:
`BRAVE_START` -> `BRAVE_RESOLVE_HOST_REQUEST_START`

Reply:
Fixed in c31dfff270216d3ebeca680888ebfa8bff1bdd75.